packages/core/src/Synchronizer.ts

```diff
@@ -316,6 +321,41 @@ export class Synchronizer extends EventEmitter {
         })
     }
 
+    get blocks() {
+        return this._blocks
+    }
+
+    get blockEnd() {
+        return this._blockEnd
+    }
+
+    insert(block: Promise<any>) {
+        block.then((r) => {
+            this.promises.push(r)
+        })
+    }
+
+    update() {
```

Comment:
same here, the logic can be put in the `loadBlocks` function

Reply:
moved logic to `loadBlocks`